### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Mmalumbo**
Signed-off-by: mmalumbo <mmalumbo@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
-# adx-products-populator
+# ADX Products Populator
+
+This is a script that loads products into ADX from a CSV file.
+
+## Dependencies
+
+The mediator uses a number of projects and dependencies to work properly:
```
Comment (liwoo) [inline README.md]: the Script

File: src/saveDataElements.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  dataElementCode: string,
+  openLMISCode: string
+): Promise<void> => {
+  const product = new Products();
+  product.productCode = productCode;
+  product.openLMISCode = openLMISCode;
+  product.dataElementCode = dataElementCode;
+
+  const savedProduct = await connection.getRepository(Products).save(product);
+  console.log(savedProduct);
```
Comment (liwoo) [inline src/saveDataElements.ts]: use Winston for logging instead

File: README.md (hunk 1 above)
Comment (Mmalumbo) [reply README.md]: i made the change.

File: src/saveDataElements.ts (hunk 2 above)
Comment (Mmalumbo) [reply src/saveDataElements.ts]: started logging with winston.

**Pull request merged**